### ISSUE TITLE
Automod message approval no longer sends the approved message visually

### ISSUE THREAD

**Issue by Felanbird** [Chatterino/chatterino2#2790]
**Describe your issue**

When you click the "Allow" button for an automodded message, the message in question no longer appears visually below the "permitted term" message, representing that the message was sent, although it did before.

Seems to be the only issue from the forced migration via #2779, every other aspect of Automod I've tested seems to work correctly.

**OS and Chatterino Version**
win10 2db50f1

**Comment by Felanbird** [comment]
@apa420 if you don't mind, I know you handle all the automod fun

**Comment by apa420** [comment]
I don't know what changed in the api endpoints so I can't really help here.
Also can you more visually describe your issue (preferably with an image).
Is it that the `permitted term x` system message no longer gets sent in chat?
It might also just be an issue that pubsub is slower/faster than the acceptance of the message after the endpoint change.

**Comment by Felanbird** [comment]
> I don't know what changed in the api endpoints so I can't really help here.
> Also can you more visually describe your issue (preferably with an image).
> Is it that the `permitted term x` system message no longer gets sent in chat?
> It might also just be an issue that pubsub is slower/faster than the acceptance of the message after the endpoint change.

https://i.imgur.com/Bi4gjny.gif

The line in question being missing is seeing "bitch" below the approval message, like this
![autmod issue](https://user-images.githubusercontent.com/41973452/118818532-6c781380-b882-11eb-9c78-69d99e0dedb4.png)

**Comment by Felanbird** [comment]
As an added note, while trying to replicate the issue for this gif, I noticed that pre existing message approvals, (terms that have been previously approved), when re-approved, don't return a "xx permitted term" message.

That hard smells like a twitch issue though since nothing is actually being approved, despite the fact that you are having to allow the message again, but because of the fact that the approved message is missing(see above), you are given the impression that nothing happens when you press the button.

**Comment by pajlada** [comment]
See https://github.com/Chatterino/chatterino2/pull/2779#issuecomment-840899320

tl;dr: We are aware, but this issue is a good reminder.

**Comment by pajlada** [comment]
Looks like this is fixed
